Sources/Bot/Models/RequiredStatusCheck.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+public struct RequiredStatusChecks: Decodable, Equatable {
+    public let strict: Bool // Require branches to be up to date before merging?
+    public let contexts: [String] // Names of status checks marked as Required
```

Comment:
why name it `contexts`? If this is based on Github API names maybe better to hide these details into CodingKeys?